giskard_vision/landmark_detection/dataloaders/loaders.py

```diff
@@ -8,9 +8,19 @@
 from giskard_vision.core.dataloaders.meta import MetaData
 from giskard_vision.core.dataloaders.tfds import DataLoaderTensorFlowDatasets
 from giskard_vision.core.dataloaders.utils import flatten_dict
+from giskard_vision.core.detectors.base import IssueGroup
 
 from .base import DataLoaderBase
 
+EthicalIssueMeta = IssueGroup(
+    "Ethical Metadata",
```

Comment:
let's remove Metadata from here, and as we discussed let's remove the ethnicity bias_detector, head_pose_detector, etc. as these now are redundant. 

the blurring, color, resize will be also refactored once we introduce these as default metadata